runtime/src/main/java/org/corfudb/runtime/collections/CMap.java

```diff
@@ -0,0 +1,22 @@
+package org.corfudb.runtime.collections;
+
+import org.corfudb.annotations.Accessor;
+
+import javax.annotation.Nonnull;
+import java.util.Map;
+
+/**
+ * Created by box on 1/25/18.
+ */
+public interface CMap<K, V> extends Map<K, V>{
+    @Accessor
+    public @Nonnull
+    default TNode<K, V>[] getEntries() {
```

Comment:
Move this to the corfutable implementation

Reply:
Just use ImmutableMap.copyOf(map). Or ImmutableSet.copyOf(entrySet());